src/Adapter/DeliveryOptions/AbstractShipmentOptionsAdapter.php

```diff
@@ -84,18 +89,27 @@ public function getInsurance(): ?int
         return $this->insurance;
     }
 
+    /**
+     * @return string|null
+     */
+    public function getLabelDescription(): ?string
```

Comment:
Waar ga je dit gebruiken?

Reply:
Uhm, in de plugins :P Net als de andere functies hier